Sources/S3Store/S3Store.swift

```diff
@@ -67,10 +67,11 @@ extension S3Store {
 
         public init(bucket: String, path: String) {
             self.bucket = bucket
-            self.path = path
+            self.path = path.droppingLeadingSlashes
         }
 
-        public var url: String { "s3://\(bucket)/\(path)" }
+        public var objectUrl: String { "https://\(bucket).s3.\(S3Store.region).amazonaws.com/\(path)" }
+        public var s3Uri: String { "s3://\(bucket)/\(path)" }
```

Comment:
These now match the terms in the S3 info panel:

![CleanShot 2023-07-13 at 08 00 57@2x](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/assets/65520/e1539690-cb27-4358-ad25-346ae5989e20)

Note that you had `s3-us-east-2.amazonaws.com`, which I've adjusted to `s3.us-east-2.amazonaws.com` (note the `.` after `s3`). Both work, this just matches what's in the info panel.

Reply:
That's great, thanks.

This is one thing I was going to discuss with you this morning. I also had to apply a policy to the bucket, but let's chat about it.